src/Sentry.Bindings.Android/Sentry.Bindings.Android.csproj

```diff
@@ -1,9 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
some BOM or setting in your IDE is adding these to the diff, please address that

Reply:
i'm just gonna be more meticulous with git staging next time